test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -695,26 +701,35 @@ func startRealMasterOrDie(t *testing.T, certDir string) (*allClient, clientv3.KV
 				t.Log(err)
 			}
 
-			time.Sleep(100 * time.Millisecond)
+			time.Sleep(time.Second)
 		}
 	}()
 
-	if err := wait.PollImmediate(100*time.Millisecond, 10*time.Second, func() (done bool, err error) {
+	if err := wait.PollImmediate(time.Second, time.Minute, func() (done bool, err error) {
```

Comment:
this is insane startup logic for a single test 😱  Compare cmd/kube-apiserver/app/testing/server_test.go.

Reply:
It is a necessary evil to get into the guts of the master which are not exposed by the default testing bits (and its not like anyone else cares about direct etcd access).  This is completely based off of `TestAggregatedAPIServer`.

Reply:
not a blocker for this PR, just some observation.